dev_guide/builds.adoc

```diff
@@ -341,19 +345,12 @@ This ensures that the triggered build uses the new image that was just pushed to
 the repository, and the build can be re-run anytime with exactly the same
 inputs.
 
-For link:../architecture/core_objects/builds.html#sti-build[STI type builds],
-the field that is matched and replaced is the `*stiStrategy.image*` field. For
-Docker builds, the field is `*dockerStrategy.baseImage*`. For custom builds, the
-`*customStrategy.image*` field is updated.
```

Comment:
So this paragraph is no longer relevant? I mainly ask because I noticed the example of the resulting build's `"strategy"` stanza (up on L330) stayed the same.